### PR TITLE
Add language editing

### PR DESCRIPTION
**Pull request by hidde**
Updates compared to previous versions of add languages:

* Uses new interaction pattern as discussed with @mbransn 
* Like the latest version of `EditAccounts`, it now just uses one `languages` object for all the things
* Restores edit button in empty card

### CODE REVIEW

File: src/components/profile/edit/EditLanguages.vue [hunk 1]
```diff
@@ -53,56 +59,78 @@ export default {
...
+      if (this.newLanguage.length > 0) {
+        this.addLanguage(this.newLanguage);
+        this.addingLanguage = false;
+      } else {
+        this.$refs.inputLanguage.focus();
+      }
+    },
     addLanguage(language) {
-      const id = Object.keys(this.languages).length + 1;
+      const id = this.languages.values[this.languages.values.length] + 1;
```
Comment (hidde) [inline src/components/profile/edit/EditLanguages.vue]: @fiji-flo Could you please help me with this? 🙏 The object needs to have a number for `k` that is higher than the previous one. I came up with this, but it breaks when user deletes a language, as then this number will be too low resulting in accidentally overwriting an existing entry.

Or do perhaps know of a smarter way to ensure the `k`'s in `language` are just always 1, 2, 3, 4… ?

Comment (Gregoor) [reply src/components/profile/edit/EditLanguages.vue]: `const maxKey = Object.keys(this.languages.value).reduce((max, key) => Number(key) > max ? Number(key) : max, 0)`

Comment (hidde) [reply src/components/profile/edit/EditLanguages.vue]: thanks so much @Gregoor ! (for reference: we decide as long as new keys are higher than any existing key, that's fine)

**Comment by hidde** [comment]
Fixed merge conflict. This is now ready for review.

**Pull request merged**